stencil-library/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file. Dates are d
 
 Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
+#### 0.6.2
+> 24 April 2023
+
+- Fix broken prop controller in Storybook UI for card-form component
+- rename variable `validationStrategy` and attr `validation-strategy` to `validationMode` and `validation-mode` respectively
```

Comment:
Please change `attr` to `prop`, just to be extra clear that it is a stencil prop